Sites/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>GD2 PR2</title>
+    <link rel="stylesheet" href="Styles.css">
+</head>
+
+<body>
+    <div>
+        <div class="container clearfix">
```

Comment:
1. There don't appear to be any floated elements in this layout, so you can remove the `clearfix` class here.
1. This `div` isn't closed at the end of the markup ⬇️⬇️⬇️